packages/nestjs-firebase-admin/lib/modules/FirebaseAdminModule.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable import/no-unresolved */
+import { DynamicModule, Module } from '@nestjs/common';
+import { AppOptions } from 'firebase-admin';
+import { initializeApp } from 'firebase-admin/app';
+import { AppCheck, getAppCheck } from 'firebase-admin/app-check';
+import { Auth, getAuth } from 'firebase-admin/auth';
+import { Firestore, getFirestore } from 'firebase-admin/firestore';
+import { getInstallations, Installations } from 'firebase-admin/installations';
+import { getMachineLearning, MachineLearning } from 'firebase-admin/machine-learning';
+import { getMessaging, Messaging } from 'firebase-admin/messaging';
+import { getProjectManagement, ProjectManagement } from 'firebase-admin/project-management';
+import { getRemoteConfig, RemoteConfig } from 'firebase-admin/remote-config';
+import { getSecurityRules, SecurityRules } from 'firebase-admin/security-rules';
+import { getStorage, Storage } from 'firebase-admin/storage';
+
+import { AppAsyncOptions } from './AppAsyncOptions';
+
+type FirebaseProviderType =
```

Comment:
This type is being used at `FirebaseAdminModule.forRoot`, so it should be exposed in order to expose it at `index` module

Reply:
Same https://github.com/cuaklabs/nestjs-meow/pull/1#discussion_r755593904